src/mlmc/estimator.py

```diff
@@ -301,6 +301,27 @@ def plot_var_compare(self, nl):
     def plot_var_var(self, nl):
         self[nl].plot_bootstrap_var_var(self._moments_fn)
 
+    def fine_coarse_violinplot(self):
```

Comment:
Pandas is a nontrivial dependency that can make compatibility issues. Possibly move the creation of the frame from the sample storage into `tools/plot.py` as well.

Reply:
sample storage doesn't create data frame, it is done here in estimator, so I think there is no use in moving it from estimator to tools/plot.py

Reply:
The data frame is created only for plotting so it is part of plotting and thus should be in the `tools/plot.py`.